### PR TITLE
Gradle Plugins Whip README

### PR DESCRIPTION
**Pull request by codyhoag**
cc @Ithildir

### CODE REVIEW

**Comment by jhinkey** [comment]
Started reviewing. Thanks.

File: modules/sdk/gradle-plugins-whip/README.markdown [hunk 1]
```diff
@@ -37,8 +37,9 @@ repositories {
 }
 ```
 
-By default, Whip is automatically applied to all the tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
-If a task has Whip applied and it is [enabled](#enabled), then Whip is configured as a Java Agent.
+By default, Whip is automatically applied to all tasks of type [`Test`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html).
+If a task has Whip applied and is [enabled](#enabled), then Whip is configured
+as a Java Agent.
```
Comment (jhinkey) [inline modules/sdk/gradle-plugins-whip/README.markdown]: @codyhoag , @Ithildir - Is it the task or Whip that must be enabled for Whip to be configured as a Java Agent?

My guess is that Whip must be enabled, but I wanted to check. Thanks.

Comment (codyhoag) [inline modules/sdk/gradle-plugins-whip/README.markdown]: Ah, good catch. I think Whip must be enabled, since the `enabled` link is for a Whip property.

Comment (Ithildir) [inline modules/sdk/gradle-plugins-whip/README.markdown]: @codyhoag is right, the task must have Whip applied, and the `task.whip.enabled` property must be `true`.

**Comment by jhinkey** [comment]
Sent https://github.com/brianchandotcom/liferay-portal/pull/42026

**Pull request closed without merging**